moped-editor/src/views/projects/projectView/ProjectSummaryEditMap.js

```diff
@@ -108,11 +118,14 @@ const ProjectSummaryMap = ({
         setFeatureCollection={setEditFeatureCollection}
       />
       {error && (
-        <Container>
-          <Alert severity="error">
-            The map edit failed to save. Please try again.
-          </Alert>
-        </Container>
+        <Alert className={classes.mapAlert} severity="error">
+          {mapErrors.failedToSave}
+        </Alert>
+      )}
+      {!areMinimumFeaturesSet && (
+        <Alert className={classes.mapAlert} severity="error">
+          {mapErrors.minimumLocations}
+        </Alert>
```

Comment:
Great feedback, and it is also cool that the save icon is disabled to prevent bugs!